src/components/DocumentationTopic/PrimaryContent.vue

```diff
@@ -65,6 +66,11 @@ export default {
         Object.prototype.hasOwnProperty.call(SectionKind, kind)
       )),
     },
+    enableMinimized: {
```

Comment:
I think we should try to avoid passing this flag to lots of subcomponents if possible and try to keep it to the one main one in the documentation topic. It would be nice if most of the specialized classes/styles associated with that flag could be easily colocated in the same component and it would be easier to maintain.